compiler/src/iree/compiler/Dialect/Flow/Transforms/FusionOfTensorOps.cpp

```diff
@@ -197,6 +198,9 @@ static FailureOr<unsigned> fuseMultiUseProducers(Operation *funcOp,
   unsigned numCandidates = 0;
   OpBuilder builder(context);
   funcOp->walk<WalkOrder::PreOrder>([&](linalg::GenericOp genericOp) {
+    if (!isNonNullAndOutsideDispatch(genericOp)) {
```

Comment:
nit: I'm wondering if we should use the same `fuseElementwiseOpsControlFn` to also control the multi-use fusion?

Reply:
I havent looked into that, but at the high level I think the requirements are a bit different for non-multi-use fusion and multi-use fusion....